### PR TITLE
Personalized health technologies workshop

### PR DESCRIPTION
**Pull request by bethcg**

### CODE REVIEW

File: blog/2024-08-30-PHRT-workshop/index.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+and reproducible (RTR) fashion and achieve FAIR data management (ensuring that all
+data is Findable, Accessible, Interoperable, and Reusable).
+
+The Gatekeeper project aims to address this need within the Biomedical domain by
+leveraging the open-source Renku platform, which connects the ecosystem of data, code,
+and compute to empower researchers to build collaborative communities.
+
+In this workshop, the participants learned about some of the tools and techniques
+important for FAIR data management and RTR research, and learned how to seamlessly
+integrate their own research data and code into the Renku platform. Thus, participants
```
Comment (lokijuhy) [inline blog/2024-08-30-PHRT-workshop/index.mdx:47]:
```suggestion
integrate their own research data and code into the Renku platform. In addition, participants
```

File: blog/2024-08-30-PHRT-workshop/index.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+provided valuable feedback about new requirements and necessary extensions.
```
Comment (lokijuhy) [inline blog/2024-08-30-PHRT-workshop/index.mdx:48]: Want to add a line thanking our workshop participants?

**Review by lokijuhy — CHANGES_REQUESTED** [review]
Haha did you use the abstract I tried to re-write for them but they didn't accept? Glad it's getting used after all! 😆 

Minor suggestions...

File: blog/2024-08-30-PHRT-workshop/index.mdx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  workshop "Empowering FAIR data and reproducible research" within the 
+  [Personalized Health Technologies 2024 event](https://personalizedhealth.ch/#overview).
+  Participants could learn about the Gatekeeper project, where Renku is leveraged
+  for empowering researchers to build collaborative communities within the biomedical
+  domain by bringing together data, code and compute.
+---
+
+![Renku in PHRT 2024](./image_1.png)
+
+<!-- truncate -->
```
Comment (leafty) [inline blog/2024-08-30-PHRT-workshop/index.mdx:22]: Truncate after some text, this is not engaging.

**Review by leafty — APPROVED** [review]

**Pull request merged**